### PR TITLE
Jk product popularity

### PR DESCRIPTION
**Pull request by juliakimchung**
# Description
created cli method to retrieve the product name, number of orders per the product, number of customers purchased the product and total revenue per product
commented out the methods to get total orders, customer num, total revenue to get the command line test work 
# Related Ticket(s)
#15 #18 #19

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x ] I certify that all existing tests pass- only command line test passed

## Documentation
all docstrings added

### CODE REVIEW

File: cli/active_customer_cli.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import sys
 sys.path.append("../")
-# from db.customer_data_man import CustomerDataMan
+from db.customer_db_interactor import Customer_db
```
Comment (samphillips1879) [inline cli/active_customer_cli.py]: Reformat of all_customers looks good!

Comment (samphillips1879) [inline cli/active_customer_cli.py:3]: Reformat of all_customers looks good!

File: db/product_db_interactor.py [hunk 2]
```diff
@@ -11,7 +13,7 @@ def save_product(self, product):
         """
         This is a method to save product data
         """
-        with sqlite3.connect('bangazon.db') as proc:
+        with sqlite3.connect(configuration.database_path) as proc:
```
Comment (samphillips1879) [inline db/product_db_interactor.py:16]: paths look good

**Comment by pinkston4** [comment]
👍 succefully created customer and viewed all customers in the cli

**Review by samphillips1879 — COMMENTED** [review]
Customer creation and selection works perfect on my machine :1st_place_medal:  lots of commented out code, but if that's necessary until we can tie in the needed modules then I say 👍

**Comment by samphillips1879** [comment]
Pull request should show a relation to tickets #18 and #19 given the customer functionality you added.

**Pull request merged**